README.md

```diff
@@ -43,6 +43,10 @@ npm install minargs
 - `recursive` (`Boolean`)
   - Default: `false`
   - Define whether or not to end parsing when a bare `--` marker is found
+🤔 Is `recursive` really essential? it seems to me that it can be fairly
+trivial for the user to be recursively parsing the remainder in case they need
+it, IMO having in mind a lib that strives to be minimal, this part can be
+removed for the sake of simplicity.
 
```

Comment:
I originally had this removed as you're right that it is trivial to add (although it's definitely nicer to have baked in via option & consistent with what we've done for `mkdir`/`rmdir`)